contracts/okp4-objectarium/src/compress.rs

```diff
@@ -77,21 +88,58 @@ fn snappy_decompress(data: &[u8]) -> Result<Vec<u8>, CompressionError> {
     Ok(writer)
 }
 
+// lzma_compress returns the LZMA compressed data.
+#[inline]
+fn lzma_compress(data: &[u8]) -> Result<Vec<u8>, CompressionError> {
+    let mut reader = io::Cursor::new(data);
+    let mut writer = Vec::new();
+    {
+        lzma_rs::lzma_compress(&mut reader, &mut writer)?;
+    }
+    Ok(writer)
+}
+
+// lzma_decompress returns the LZMA decompressed data.
+#[inline]
+fn lzma_decompress(data: &[u8]) -> Result<Vec<u8>, CompressionError> {
+    let mut reader = io::Cursor::new(data);
+    let mut writer = Vec::new();
+    {
+        lzma_rs::lzma_decompress(&mut reader, &mut writer)?;
+    }
```

Comment:
No need for parenthesis.

```suggestion
        lzma_rs::lzma_decompress(&mut reader, &mut writer)?;
```